go.mod

```diff
@@ -12,6 +14,9 @@ replace (
 	github.com/rancher/rancher/pkg/apis => ./pkg/apis
 	github.com/rancher/rancher/pkg/client => ./pkg/client
 
+	//github.com/rancher/shepherd => /Users/mspencer/Workspace/shepherd
+	github.com/rancher/shepherd => github.com/mspencer87/shepherd v0.0.0-20240426094202-bdf694705c76
```

Comment:
Just a reminder that once the shepherd PR is merged, this will need to point to the updated `main` branch commit and not your forked repo.